src/shared/bond-util.h

```diff
@@ -0,0 +1,106 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+#pragma once
+
+#include <netinet/in.h>
+#include <linux/if_bonding.h>
+
+#include "in-addr-util.h"
+
```

Comment:
in-addr-util.h is still included here, should be macro.h

Reply:
oops sry ! I missed it.